src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -193,6 +193,11 @@
         --auto-regenerate-key --regeneration-period P90D  --resource-id $id
 """
 
+helps['keyvault storage remove'] = """
+type: command
+short-summary: Remove a Key Vault managed Azure Storage Account and all associated SAS definitions.
```

Comment:
Storage Account -> storage account

Reply:
@qwordy I copied this from PowerShell docs for consistency: https://docs.microsoft.com/en-gb/powershell/module/az.keyvault/remove-azkeyvaultmanagedstorageaccount?view=azps-3.1.0&viewFallbackFrom=azps-1.0.0

Reply:
It's not a big problem.
This is the first result in search engine.
https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview